pom.xml

```diff
@@ -20,6 +20,15 @@
         </maven.checkstyle.plugin.configLocation>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+        </dependency>
```

Comment:
As for me, you can delete this dependency

Reply:
done